docs/conf.py

```diff
@@ -188,9 +188,7 @@
         "json_url": json_url,
         "version_match": version_match,
     },
-    "navigation_with_keys": False,
```

Comment:
False is now the default in `theme.conf` so we don't need to specify it in our own docs (or in any tests) just to avoid the deprecation warning